app/models/form/sales/pages/person2_known.rb

```diff
@@ -7,9 +7,9 @@ def initialize(id, hsh, subsection)
     @description = ""
     @subsection = subsection
     @depends_on = [
-      { "hholdcount" => 2 },
-      { "hholdcount" => 3 },
-      { "hholdcount" => 4 },
+      { "hholdcount" => 2, "details_known_1" => 1  },
+      { "hholdcount" => 3, "details_known_1" => 1  },
+      { "hholdcount" => 4, "details_known_1" => 1  },
```

Comment:
Hmm, would we still want to ask this question even if we don't know the details for person 1? 🤔

Reply:
I don't think so - see https://miro.com/app/board/uXjVPcAfS04=/

Reply:
Hmm, we display it for the lettings, so that might be a bit inconsistent, maybe worth checking with design team?
<img width="733" alt="image" src="https://user-images.githubusercontent.com/54268893/207115794-14aa1175-8f6c-4eda-8e6b-09a5b3725861.png">